build.sbt

```diff
@@ -75,6 +103,10 @@ lazy val publishSettings = Seq(
       Some("releases"  at nexus + "service/local/staging/deploy/maven2")
   },
   publishArtifact in Test := false,
+  pgpSecretRing := file("local.secring.gpg"),
```

Comment:
Should we add `local.*` into `.gitignore`?

Reply:
Also, what's interesting is I have gpg 2.2.10 installed locally and it doesn't use the `secring.gpg` file (according to this [doc](https://www.gnupg.org/documentation/manuals/gnupg/GPG-Configuration.html)):

> A secret keyring as used by GnuPG versions before 2.1. It is not used by GnuPG 2.1 and later. I'm not sure which version of gpg utility sbt-gpg depends on so perhaps I should downgrade to make sure I generate both files.

Reply:
I decided to use the older version of GPG I installed via `brew`. I'm pretty sure that version works just fine with `sbt-gpg` plugin:

```
$ gpg --version
gpg (GnuPG) 1.4.18
Copyright (C) 2014 Free Software Foundation, Inc.
```